post_office/management/commands/send_queued_mail.py

```diff
@@ -36,25 +28,5 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-        logger.info('Acquiring lock for sending queued emails at %s.lock' %
-                    options['lockfile'])
-        try:
-            with FileLock(options['lockfile']):
-
-                while 1:
-                    try:
-                        send_queued(options['processes'],
-                                    options.get('log_level'))
-                    except Exception as e:
-                        logger.error(e, exc_info=sys.exc_info(),
-                                     extra={'status_code': 500})
-                        raise
-
-                    # Close DB connection to avoid multiprocessing errors
-                    connection.close()
-
-                    if not Email.objects.filter(status=STATUS.queued) \
-                            .filter(Q(scheduled_time__lte=now()) | Q(scheduled_time=None)).exists():
-                        break
-        except FileLocked:
-            logger.info('Failed to acquire lock, terminating now.')
\ No newline at end of file
+        logger.info('Acquiring lock for sending queued emails at %s.lock', options['lockfile'])
```

Comment:
Can we move this logging call to inside `send_all_in_queue`? So this message will also appear when the lock is being acquired.

Reply:
done